### PR TITLE
Demo server yml config and guide update

### PR DESCRIPTION
**Pull request by CarlosNZ**
As discussed in Zoom call

### CODE REVIEW

File: documentation/Demo-Server-Guide.md [hunk 1]
```diff
@@ -184,9 +185,9 @@ sudo docker exec -ti mflow-on-8000_app_1 cat /var/log/application_manager/graphi
 
 ```bash
 # to stop
-sudo docker-compose --project-name 'mflow-on-8000' stop
+sudo docker-compose -E --project-name 'mflow-on-8000' stop
 # to remove (when new version is out)
-sudo docker-compose --project-name 'mflow-on-8000' down
+sudo docker-compose -E --project-name 'mflow-on-8000' down
```
Comment (nmadruga) [inline documentation/Demo-Server-Guide.md:190]: it doesn't need the PORT_APP and PORT_DASH passed as well?

Comment (CarlosNZ) [reply documentation/Demo-Server-Guide.md:190]: Yeah, you're right. Stand by...

Comment (CarlosNZ) [reply documentation/Demo-Server-Guide.md]: Done 5d8ddac1

**Review by nmadruga — APPROVED** [review]
LGTM

Thanks Carl : )

**Pull request merged**